### PR TITLE
Fix errors in deploying dualtor-120.

### PR DESCRIPTION
**Pull request by bingwang-ms**
### Description of PR

Summary:
This PR fixed several errors detected when deploying ```dualtor-120```.
1. Add missing ```dualtor-120``` in ```ansible/veos``` and ```ansible/veos_vtb```
2. Shorten name template for interface to contain more digits.
3. Add an argument ```topology``` for ```vm_topology``` library.

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
This PR is to fix errors detected when deploying dualtor-120.

#### How did you do it?
Please see code.

#### How did you verify/test it?
Verified by deploying the newly defined ```dualtor-120```.

#### Any platform specific information?
No.

#### Supported testbed topology if it's a new test case?
No.

### CODE REVIEW

File: ansible/roles/vm_set/library/vm_topology.py [hunk 1]
```diff
@@ -114,8 +114,11 @@
 OVS_FP_BRIDGE_TEMPLATE = 'br-%s-%d'
 OVS_FP_TAP_TEMPLATE = '%s-t%d'
 OVS_BP_TAP_TEMPLATE = '%s-back'
-INJECTED_INTERFACES_TEMPLATE = 'inje-%s-%d'
-MUXY_INTERFACES_TEMPLATE = 'muxy-%s-%d'
+# The name of interface name must be less than 15 bytes.
+# As a result, interface name like 'inje-vms21-6-132' will be invalid and rejected by kernel.
+# Therefore, the name template is shorten to 3 bytes to contain more digits (3 in max)
+INJECTED_INTERFACES_TEMPLATE = 'inj-%s-%d'
+MUXY_INTERFACES_TEMPLATE = 'mux-%s-%d'
```
Comment (yxieca) [inline ansible/roles/vm_set/library/vm_topology.py:121]: This is a disruptive change. This change will cause interface name change, so testbed deployed before this change cannot be removed after.
How do we deal with that?

Comment (bingwang-ms) [reply ansible/roles/vm_set/library/vm_topology.py:121]: Thanks for the good catch. I also noticed that some other components that depend on the interface name or bridge name will fail.

**Comment by bingwang-ms** [comment]
Update. I implemented an ```adaptive_name``` module function to calculate interface and bridge name to ensure a valid length. And keep back compatible at the same time.

**Review by lolyu — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Pull request merged**